site-src/_includes/epp-latest.md

```diff
@@ -30,3 +30,13 @@
       --version $IGW_CHART_VERSION \
       oci://us-central1-docker.pkg.dev/k8s-staging-images/gateway-api-inference-extension/charts/inferencepool
       ```
+
+=== "NGINX Gateway Fabric"
+
+      ```bash
+      export IGW_CHART_VERSION=v1.0.2
```

Comment:
`IGW_CHART_VERSION` is not set here. This _includes is referenced by the latest/main quickstart guide which uses the [latest rolling published EPP chart](https://console.cloud.google.com/artifacts/docker/k8s-staging-images/us-central1/gateway-api-inference-extension/charts%2Finferencepool), i.e. `main` branch.